lib/miq_tools_services/minigit.rb

```diff
@@ -153,19 +144,11 @@ def mergeable?(branch = nil, into_branch = "master")
       end
     end
 
-    def update_pr_branch(branch = nil, remote = "upstream")
-      create_or_update_pr_branch(branch || current_branch, remote)
-    end
-
-    def create_pr_branch(branch, remote = "upstream")
-      create_or_update_pr_branch(branch, remote)
-    end
-
-    private
-
-    def create_or_update_pr_branch(branch, remote)
-      fetch("-fu", remote, "refs/pull/#{pr_number(branch)}/head:#{branch}")
-      reset("--hard")
+    def ensure_fetch_refs
```

Comment:
I didn't want to be specific about PRs here, because I assume we're going to want to ad MRs in the near future